software/py/blood_graph.py

```diff
@@ -3,68 +3,292 @@
 #
 #   vanilla_core execution visualizer.
 # 
-#   input: vanilla_stall_trace.log
+#   input: vanilla_operation_trace.log
```

Comment:
.log or .csv?

Reply:
or .trace

Reply:
changed to csv, to be aligned with the vanilla_stats.csv. Thanks!